commitlintplugins.test.ts

```diff
@@ -5,6 +5,13 @@ function runCommitLintOnMsg(inputMsg: string) {
 }
 
 
+test('allow-PascalCase-after-colon-in-title', () => {
+    let commitMsgWithPascalCaseAfterColon = "foo/bar: PascalCase bla blah";
```

Comment:
2 nits here:
- The simpler the test, the better. In this sense, there's no need to add a sub-area, so let's replace `foo/bar` with `foo` (this way, we also reduce the possible overlap between tests; i.e. imagine that some future dev inverts, by mistake, the rule that prefers slash over backslash, then the consequence of it would be to have this test broken even if it's unrelated to that; so if you replace "foo/bar" with "foo" and this happens, only the tests related to the backslash rule would fail).
- Let's use for this test the exact same message that we had in the TODO (because it was a realistic message that could happen in real life; this way it is more understandable why we need this exception).

Reply:
Done!